docs/content/deployment/guides/aws.mdx

```diff
@@ -75,7 +75,19 @@ run_launcher:
 
 ### Customizing CPU and memory in ECS
 
-You can use job tags to customize the CPU and memory of every run for a job:
+You can set the `run_launcher.config.run_resources` field to customize the default memory and CPU resources for Dagster runs. For example:
+
+```yaml
+run_launcher:
+  module: "dagster_aws.ecs"
+  class: "EcsRunLauncher"
+  config:
+    run_resources:
+      cpu: 256
+      memory: 512
```

Comment:
Should we specify the size/measurements for these values? Ex: GB?

Or should we point them to the AWS docs and say something like "Refer to the [AWS documentation]() for valid sizes/values/etc"?

Reply:
we do that below on line 111 at the bottom of this section - we could move it earlier if we are worried people will miss it now that there's more stuff here